.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do gitignore trzeba będzie wrzucić te od intelij pliki, ale to moja wina, że nie zrobiłam tego od razu :P

Reply:
Znaczy ja tego nie byłem pewny, bo nie wiedziałem co jest potrzebne, bałem się ruszać :P